utils/zerovec/src/lib.rs

```diff
@@ -556,3 +518,41 @@ pub use zerovec_derive::make_ule;
 /// ```
 #[cfg(feature = "derive")]
 pub use zerovec_derive::make_varule;
+
+#[cfg(test)]
```

Comment:
question: Why did you move this?

Reply:
I think clippy lints about the location of test modules now, I recall seeing that lint PR go by

Reply:
I am but a humble servant to our clippy overlord.